bond_app/jwt_token.py

```diff
@@ -11,5 +12,7 @@ def __init__(self, encoded_str, user_name_path_expr):
         :param user_name_path_expr: forward slash delimited path to user name within JWT
         """
         self.raw_dict = jwt.decode(encoded_str, verify=False)
+        # See https://broadworkbench.atlassian.net/browse/CA-1341, needed for dpath to work when dict has "" keys
```

Comment:
(As a note, this is unassigned right now, so should probably be assigned to you?)